packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -24,12 +24,12 @@ const log = Log.module('MonacoUtils');
 
 declare global {
   interface Window {
-    MonacoEnvironment: Environment;
+    MonacoEnvironment?: Environment;
   }
 }
 
 window.MonacoEnvironment = {
-  getWorker(workerId, label) {
+  getWorker() {
     return new EditorWorker();
```

Comment:
This will probably need to be resolved with the latest main - we downgraded monaco-editor and it should just be `return editorWorker()` (I think)